dashboard/pages/compliance.py

```diff
@@ -37,7 +37,7 @@
 
 csv_files = []
 for file in glob.glob(os.path.join(folder_path_compliance, "*.csv")):
-    with open(file, "r", newline="") as csvfile:
+    with open(file, "r", newline="", encoding="utf-8") as csvfile:
```

Comment:
Can you add the encoding in the config and the use it here? The same for the other file.

Reply:
Good catch, I´ve already done it